config/cchost_config_virtual_js.json

```diff
@@ -1,32 +1,34 @@
 {
-    "enclave": {
-      "file": "/usr/lib/ccf/libjs_generic.virtual.so",
-      "type": "Virtual"
+  "enclave": {
+    "file": "/usr/lib/ccf/libjs_generic.virtual.so",
+    "type": "Virtual"
+  },
+  "network": {
+    "node_to_node_interface": {
+      "bind_address": "172.17.0.3:8081"
```

Comment:
Should we make changes to the enclave config file as well? (ok if done separately, just raising the question)

Reply:
I'd rather tidy up these config files as part of #56